### DISCUSSION TITLE
Separating a `if` clause and a `class` declaration opening in a same line to different lines

### DISCUSSION THREAD

**Discussion by onet4** [PHP-CS-Fixer/PHP-CS-Fixer#6819] [Q&A]
## Support question

Hello,

The following code does not get formatted well with the default configuration.

```php
<?php
if (! class_exists('A', false)) : abstract class A {}
endif;  
```

It becomes
```php
<?php

if (! class_exists('A', false)) : abstract class A
{
}
endif;
```
I want it to be 
```php
<?php
if (! class_exists('A', false)) : 
    abstract class A {
    }
endif;
```

I tried adding the following rule but doesn't help.
```php
    'class_definition' => [
        'single_item_single_line' => true,
        'single_line' => true,
    ],
```

Is it possible?

**Comment by Wirone** [comment]
You should try `statement_indentation` rule. It supports alternative syntax (`if(): ... endif;`), so if it does not work in your case, it can be a bug in that fixer.

But I am not sure if that rule can insert new lines, or if it only fixes indentation in existing lines... 🤔. I am AFK now so I can't check.

**Comment by keradus** [comment]
As a design decision, I do not want PHP CS Fixer to support alternative syntax. I discourage usage of it and almost none rule supports it. (well, except of `no_alternative_syntax` ;) )
